src/sentry/migrations/0712_create_tombstone_compound_indexes.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 5.0.4 on 2024-05-02 17:26
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = True
+
+    dependencies = [
+        ("sentry", "0711_backfill_group_attributes_to_self_hosted"),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name="controltombstone",
+            index=models.Index(
+                fields=["table_name", "object_identifier"], name="sentry_cont_table_n_940c6d_idx"
+            ),
+        ),
```

Comment:
Is this going to work in a split deployment? Both of these index operations target different silos and tables